ocrd_network/ocrd_network/deployer.py

```diff
@@ -525,9 +521,8 @@ def start_native_processor_server(
 
     # TODO: No support for TCP version yet
     def start_unix_mets_server(self, mets_path: str) -> str:
-        socket_file = f'{safe_filename(mets_path)}.sock'
-        log_path = f'/tmp/{safe_filename(mets_path)}.log'
-        mets_server_url = f'/tmp/{socket_file}'
+        log_file = get_mets_server_logging_file_path(mets_path=mets_path)
+        mets_server_url = f'/tmp/{safe_filename(mets_path)}.sock'
```

Comment:
Good point.

Reply:
I think we should not use `tempfile.gettempdir()` here. The reason tmp is used is that it allows root and the "normal" user (i.e. 1000) to write and read here. The processing server and the workers (possible deployed in docker and /tmp is a volume mounted dir from the host) must be able to read and write here. I don't see any point in using tempfile. Tbh I would prefer this to be env-configurable and I would rather like to see it in /data instead of /tmp.

Reply:
@joschrew, you are right. I paid attention for the root log folder to be consistent since it will be volume mapped in the docker environment. So I did not use the temp dir there. However, I did not realize that the same issue is here for the socket files as well.

Reply:
`tempfile.gettempdir()` is `/tmp` in most cases. I agree we should make this overrideable (cf. https://github.com/OCR-D/core/pull/1111#discussion_r1356583376).

Reply:
ok, thanks for clarifying, I thought it would create `/tmp/<random-name>` ...

Reply:
@kba, `tempfile.gettempdir()` indeed creates a temporary directory under `/tmp`, so it is still `/tmp/<random-name>`

Reply:
How do you mean?

```python
>>> import tempfile
>>> tempfile.gettempdir()
'/tmp'
```

What does that return for you?

Reply:
@kba, sorry for the confusion I have created - you're right. The issue I had was related to another bug.

Reply:
Resolved in https://github.com/OCR-D/core/pull/1111/commits/464f8eaf8c2c696668699a880f885dea14847f1b